src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -31,7 +31,7 @@
     (utils/join-existing-users-string user-list)]])
 
 (defn channel-token-gating-details
-  [name token-gating emoji channel-color]
+  [name gates emoji channel-color]
```

Comment:
could you elaborate please on renaming ?

Reply:
Hm.. at the moment of renaming I had some internal reasoning.. but can't remember it)) So definetely can be rolled back)